modules/apps/frontend-css/frontend-css-cadmin-web/src/main/resources/META-INF/resources/clay_admin.scss

```diff
@@ -33,6 +33,14 @@ html#{$cadmin-selector} {
 			position: static;
 		}
 
+		&.modal-full-screen,
```

Comment:
We can remove all this here. It makes the first modal take up the whole space when there should be some space:
![modal-takes-screen](https://user-images.githubusercontent.com/788266/168667705-72cb9634-c63f-420a-98d6-a7676c3ba1af.jpg)

without modal-full-screen styles:
![modal-0000](https://user-images.githubusercontent.com/788266/168667743-c87a1991-afdc-41a4-84c0-46f90081a375.jpg)